src/m365/purview/commands/retentionevent/retentionevent-remove.spec.ts

```diff
@@ -120,18 +120,18 @@ describe(commands.RETENTIONEVENT_REMOVE, () => {
   });
 
   it('Correctly deletes retention event by id', async () => {
-    sinon.stub(request, 'delete').callsFake((opts) => {
-      if (opts.url === `https://graph.microsoft.com/v1.0/security/triggers/retentionEvents/${validId}`) {
-        return Promise.resolve();
+    sinon.stub(request, 'delete').callsFake(async (opts) => {
+      if (opts.url === `https://graph.microsoft.com/beta/security/triggers/retentionEvents/${validId}`) {
+        return;
       }
 
-      return Promise.reject('Invalid Request');
+      throw 'Invalid Request';
     });
 
     sinonUtil.restore(Cli.prompt);
-    sinon.stub(Cli, 'prompt').callsFake(async () => (
-      { continue: true }
-    ));
+    sinon.stub(Cli, 'prompt').callsFake(async () => {
+      return { continue: true };
+    });
```

Comment:
Seems like we had a little miscommunication here. I mean to use `.returns({ continue: true })` in the first place. But because this is an async function, it should be `.resolves(`. 😊